packages/@lwc/integration-karma/test/static-content/x/attribute/attribute.html

```diff
@@ -3,7 +3,12 @@
     <div data-id="dynamicAttr" data-value={dynamicAttr}></div>
     <div data-id="staticStyle" style="color: blue;"></div>
     <div data-id="dynamicStyle" style={dynamicStyle}></div>
+    <div data-id="staticEmptyStyle" style=""></div>
+    <div data-id="staticSpacesStyle" style="   "></div>
+    <div data-id="staticInvalidStyle" style="invalid"></div>
     <div data-id="staticClass" class="static class"></div>
+    <div data-id="staticSpacesClass" class="   "></div>
+    <div data-id="staticEmptyClass" class=""></div>
```

Comment:
Just out of paranoia, can we also check:

- tab character
- "boolean true" i.e. `<div class>`

Reply:
It would also be great to test these with and without scoped styles, to make sure the concatenation is done correctly. You could extend `attribute.js` and add `static stylesheets` to do this.